src/main/java/bsise/server/letter/LetterRepository.java

```diff
@@ -1,15 +1,27 @@
 package bsise.server.letter;
 
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.UUID;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface LetterRepository extends JpaRepository<Letter, UUID> {
 
     Page<Letter> findLettersByUserId(UUID userId, Pageable pageable);
     List<Letter> findTop10ByPublishedIsTrueOrderByCreatedAtDesc();
+
+    @Query(value = "SELECT * FROM letter " +
+                   "WHERE user_id = :userId AND created_at BETWEEN :startTime AND :endTime " +
+                   "ORDER BY created_at DESC LIMIT 3", nativeQuery = true)
```

Comment:
```suggestion
    @Query(value = """
                        SELECT * FROM letter
                        WHERE user_id = :userId AND created_at BETWEEN :startTime AND :endTime
                        ORDER BY created_at DESC
                        LIMIT 3
                    """,
                    nativeQuery = true)
```

텍스트 블록으로 감싸면 여러 줄의 문자열에서 발생할 수 있는 띄어쓰기 문제를 방지할 수 있습니다

Reply:
nativeQuery일 때 뿐만 아니라 jpql일때도 전부 수정하면 될까요? 다만 jpql의 경우 `\`로 줄바꿈을 명시해주어야 한다는 문제가 있어 그냥 그대로 갈까 싶기도 합니다.